torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -222,6 +223,7 @@ Tensor & copy_(Tensor & self, const Tensor & src, bool non_blocking) {
     grad_fn->src_device = src.device();
   }
   {
+    at::tracer::impl::NoTracerDispatchMode tracer_guard;
```

Comment:
The model here is something like TracingTypeManual

Reply:
agreed - I'll add a TODO here. The reason I added the guard is because these manual tracing logic doesn't have `setTracingState(null)` compared to auto-generated TraceType, I should either add `setTracingState(null)` above or use the guard, otherwise it might break old behavior.